collect_app/src/main/java/org/odk/collect/android/utilities/WidgetAppearanceUtils.java

```diff
@@ -122,6 +122,12 @@ public static int getNumberOfColumns(FormEntryPrompt formEntryPrompt, Context co
                         numColumns = Integer.parseInt(substringFromNumColumns.substring(0, substringFromNumColumns.contains(" ")
                                 ? substringFromNumColumns.indexOf(' ')
                                 : substringFromNumColumns.length()));
+
+                        if (numColumns < 1) {
```

Comment:
Although we're only officially supporting up to 10 columns, I don't think this is necessary. I think it would be more confusing than it's worth. I think we should unofficially continue supporting arbitrary values of N.

Reply:
I'm not sure... so why do you want to document that we support up to 10 columns? We didn't perform any testing and it even wouldn't make sense since devices have different screens, so I'm sure there are devices where 4 columns are the maximum value (to keep all elements visible) and also there are devices where it's possible to have 11 columns.

Reply:
Ok I thought it over and I agree it's better to allow using > 10 columns because in some cases it could be useful (if it's a big screen and options are pretty short like A, B, C, etc.) but then I don't understand why you want to document that we support up to 10 columns.

Reply:
For spec-level purity. You can see the discussion at https://github.com/XLSForm/xlsform.github.io/issues/125#issuecomment-493792324.